src/Initializer/BasicTypedefs.hpp

```diff
@@ -50,12 +50,23 @@ enum class FaceType {
   analytical = 7
 };
 
+// Once the FaceType enum is updated, make sure to update these methods here as well.
+
+// Checks if a face type is an interior face (i.e. there are two cells adjacent to it).
+constexpr bool isInteriorFaceType(FaceType faceType) {
```

Comment:
The functionality is slightly different however, namely that the periodic faces are classified as interior (since plain, DR, and periodic are often treated as interior faces throughout the code)